examples/generated-sources/dub.json

```diff
@@ -1,11 +1,7 @@
 {
 	"name": "generated-sources",
 	"description": "Example of using pre generate commands to generate source code.",
-	"sourceFiles": [ "source/test.d" ],
```

Comment:
Could you add a test with globbing as well ?
e.g.:
```json
"sourceFiles": [ "something/*.d" ]
```

And have 2 modules there, one that is already present, one that is generated afterwards.
Those two modules should have an `extern(C) void someModulePrefix_func() {}` function, that is forward declared in the main, to avoid the need for an `import` (which would then require you to use another `dub` directive).